youtube_dl/extractor/cspan.py

```diff
@@ -18,22 +18,21 @@ class CSpanIE(InfoExtractor):
     IE_DESC = 'C-SPAN'
     _TESTS = [{
         'url': 'http://www.c-span.org/video/?313572-1/HolderonV',
-        'md5': '8e44ce11f0f725527daccc453f553eb0',
+        'md5': '067803f994e049b455a58b16e5aab442',
         'info_dict': {
             'id': '315139',
             'ext': 'mp4',
             'title': 'Attorney General Eric Holder on Voting Rights Act Decision',
-            'description': 'Attorney General Eric Holder spoke to reporters following the Supreme Court decision in Shelby County v. Holder in which the court ruled that the preclearance provisions of the Voting Rights Act could not be enforced until Congress established new guidelines for review.',
+            'description': 'Attorney General Eric Holder speaks to reporters following the Supreme Court decision in [Shelby County v. Holder], in which the court ruled that the preclearance provisions of the Voting Rights Act could not be enforced.',
```

Comment:
now it can extract multiple qualities:
```
python2 __main__.py -F http://www.c-span.org/video/?313572-1/HolderonV
[CSpan] 313572: Downloading webpage
[CSpan] 315139: Downloading JSON metadata
[CSpan] 315139: Downloading XML
WARNING: ar subtitles not available for 315139
WARNING: arME subtitles not available for 315139
[info] Available formats for 315139:
format code  extension  resolution note
80-144p      mp4        144p         80k 
628-360p     mp4        360p        628k 
628-576p     mp4        576p        628k 
4794-576p    mp4        576p       4794k  (best)
```
and handle error:
```
python2 __main__.py -F http://www.c-span.org/video/?118586-1/radio-secretary-jeh-johnson-homeland-security-threats
[CSpan] 118586: Downloading webpage
[CSpan] 418793: Downloading JSON metadata
ERROR: CSpan said: Video not available at this time
```